bentoml/_internal/bento/docker/templates/base.j2

```diff
@@ -56,9 +58,11 @@ ENV PYTHONIOENCODING=UTF-8
 
 ENV PYTHONUNBUFFERED=1
 
-ENV PATH {{ expands_bento_path("env", "docker") }}:/usr/local/bin:$PATH
+ENV PATH {{ expands_bento_path("env", "docker", bento_path=bento__path) }}:/usr/local/bin:$PATH
```

Comment:
what's this line for?

Reply:
expands to the correct bento path. For operators that needs to change bento path, this ensure that the entrypoint is can be run by anyone other than root

Reply:
Should we be adding the entrypoint to $PATH?